### DISCUSSION TITLE
Welcome to lnreader Discussions!

### DISCUSSION THREAD

**Discussion by rajarsheechatterjee** [lnreader/lnreader#202] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by T4HM1D** [comment]
Just wondering what the point of the render HTML Page option is for?

**Reply by rajarsheechatterjee** [reply]
Render HTML switches to the WebView Reader which has some extra features such as custom css, volume buttons for scroll, etc.

**Comment by ShadowMonarch100** [comment]
How to read novels after bypassing cloudfare? Like tachiyomi there is no option to retry the extension after bypassing cloudfare!

**Reply by Ushsjxhs** [reply]
ايف ذهب موقع Novele ارجوكم ارجعوه

**Comment by ImMizuu** [comment]
Thank you for making this app. It really saves time and effort to look for light novels to read

**Comment by Eiozea** [comment]
hi! Can anyone add auto read next chapter?
Pleeeeease 😭
